apmproxy/receiver_test.go

```diff
@@ -195,6 +195,8 @@ func Test_handleInfoRequest(t *testing.T) {
 		req.Header.Add(name, value)
 	}
 
+	time.Sleep(5 * time.Second)
```

Comment:
Adding some sleep here made the test more stable. It was flaky under ubuntu runners.

Reply:
Thanks, I've opened https://github.com/elastic/apm-aws-lambda/issues/368 to keep track of it